File: 3-Bimestre/C++/ex-9.cpp
#include <iostream>
using namespace std;

int main() {
    int n, contador = 1, fatorial = 1;
    cout << "Digite um numero: ";
    cin >> n;

    if (n >= 0) {
        while (contador <= n) {
            fatorial = fatorial * contador;
            contador++;
        }
        cout << "Fatorial: " << fatorial;
    } else {
        cout << "Numero invalido";
    }

    return 0;
}

File: 3-Bimestre/Turma-A/ex-9.py
contador = 1
fatorial = 1

n = int(input('Digite um número inteiro positivo: '))

if n >= 0:
    while contador <= n:
        fatorial *= contador
        contador += 1
    print(f'Fatorial: {fatorial}')
else:
    print('Número inválido')
